src/ln/channelmonitor.rs

```diff
@@ -1512,6 +1570,8 @@ impl ChannelMonitor {
 				input: inputs,
 				output: outputs,
 			};
+			let predicted_weight = spend_tx.get_weight() + Self::get_witnesses_weight(&vec![InputDescriptors::RevokedOutput]);
```

Comment:
Oops, when you took my commit you re-introduced the vec-instead-of-const-array here.

Reply:
Oh my bad, was thinking your commit was in const-array-instead-of-vec